src/app/functions/server/featureFlags.test.ts

```diff
@@ -49,7 +49,7 @@ it("returns flag state if exists", async () => {
       };
     } else {
       console.debug("flag name:", name);
-      fail("Unknown flag passed to mock");
+      throw new Error("Unknown flag passed to mock");
```

Comment:
This was apparently an invalid use of `fail`, and throwing an error was preferred according to the lint message.